### DISCUSSION TITLE
What is the Kconfig CONFIG_OPENTHREAD_OTNS used for?

### DISCUSSION THREAD

**Discussion by meatherly** [openthread/openthread#8696] [Q&A] [answered]
Hi,
I noticed that in my nordic and espressif projects have a kconfig option for `CONFIG_OPENTHREAD_OTNS` (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#CONFIG_OPENTHREAD_OTNS). I know that is the OpenThread Network Simulator but not sure how this option would apply to a physical device. I was wondering if you guys could shed some light on this. Thank you.

**Comment by jwhui** [comment, marked as answer]
The OpenThread Network Simulator (OTNS) is only for simulation and does not apply to physical devices.

See the [OTNS codelab](https://openthread.io/codelabs/openthread-network-simulator) to learn more about OTNS.
